capitalize_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'decorator'
+class CapitalizeDecorator < Decorator
+  def correct_name
```

Comment:
- This method must return a maximum of ten characters. Anything longer should be trimmed
```ruby
# Inherits from Decorator
class TrimmerDecorator < Decorator
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
end
```

Reply:
OK